### ISSUE TITLE
SMS: Kanban How-To Guide

### ISSUE THREAD

**Issue by MMdeploy-test** [testOrganisationDeploy/testRepository#1]
###### THIS IS THE LIVE EDITOR FOR YOUR NEW PROJECT, USE IT TO DOCUMENT GENERAL PROJECT INFORMATION.

This template provides a simple, high-level configuration and sample data. You can extend this sample project or build your own using the same features and views, or other advanced [Business Plan](https://help.wrike.com/hc/en-us/articles/209603949-Business) features, such as [Custom Fields](https://help.wrike.com/hc/en-us/articles/209603049-Custom-Fields), [Reports](https://help.wrike.com/hc/en-us/articles/209604449-Reports-Overview), [Custom Workflows](https://help.wrike.com/hc/en-us/articles/210322785-Custom-Statuses-and-Workflows) and [Request forms](https://help.wrike.com/hc/en-us/articles/209603169-Request-Forms). 
### How-To Guide: Kanban Project
This template will help you get up and running quickly with Wrike. Use it as a reference to learn how to use Wrike to manage a Kanban project. It's ready to go. 

The guide below shows you how to: 

* Break down your work into tasks
* Use the Board View to manage your work
* Set up complex projects using Kanban boards

Follow the steps below to get your project started. When you're done you can delete this task or create a reference Folder and [move this task](https://help.wrike.com/hc/en-us/articles/210322925-Move-Tasks-Folders-and-Projects#move) there. 
#### Step 1: Break Down Your Work
Add tasks to your Kanban board 

1. Click "**+ New task**" from any column. 
1. Enter a task name. 

Don't enter specifics just yet, later you can click on any task and fill in task descriptions, schedule start and due dates, add pictures, attach files, and more. 

【 highlighted #e2bb00 】**Wrike Tip!**【 end highlighted 】 Use verbs as the first word in your task's title. It helps let you and your team know what needs to be done and sets up a good naming convention. 
#### Step 2: Manage Your Work with the Board View
Each Board View column represents a task status. This template uses a Kanban workflow, which includes five statuses: 

* **To Do:** tasks that are up next - you'll work on them after current tasks are completed 
  * Keep your To Do tasks and backlog separate, your To Do list should include tasks which you can complete with a certain time frame, for example a week
  * Prioritize tasks within each column by dragging and dropping tasks to higher or lower on the list
* **Doing:** tasks that are in progress 
  * Use this status for tasks that you're currently working on (be honest!)
  * Gain focus by minimizing the number of tasks performed in parallel 
* **Done:** completed tasks
* **Backlog:** deferred or on hold tasks
* **Cancelled:** tasks that are no longer part of your work but which aren't ready to be deleted

Drag and drop tasks across columns to change their status. 

![](/attachments/72668979/kanban.png)

【 highlighted #e2bb00 】**Wrike Tip!**【 end highlighted 】 By default, Board view displays the first image in the task on the card. You can turn this off using three dots menu on the board toolbar. Find more information on how to use and customize Board View in our [Help Center article](https://help.wrike.com/hc/en-us/articles/115000193205-Board-View). 
#### Step 3: Set Up Complex Projects Using Kanban Boards
Manage a complex project using the Kanban principles of visual management. 

1. Create Subfolders for each phase of your complex project. 
  1. Right click on your Project
  1. Hover over Add and select Folder
  1. Enter a Folder name
  1. Select the Board View as the default view

![](/attachments/72668980/folders.png)

Each of your Folders acts as a new board within a large Project. 

【 highlighted #e2bb00 】**Wrike Tip!**【 end highlighted 】 In your main project click the menu icon and select "Show Subfolders" see tasks from Subfolders and get an Kanban view across all phases of a complex Project. 

![](/attachments/72668981/show_subfolders.png)

┆Author: Mick Mal
┆Attachments: <a href="https:&#x2F;&#x2F;storage.www.wrike.com&#x2F;extfile&#x2F;eyJ0IjoiZmExIiwiYWxnIjoiSFMyNTYiLCJjdHkiOiJhcHBsaWNhdGlvblwvanNvbiJ9.eyJhIjoyMzQ5MjAzLCJyIjoiVVMiLCJleHAiOjMyNTAzNzA4ODAwLCJpYXQiOjE1Mzg3NTIwMTQsImp0aSI6IjcyNjY4OTc5In0.RwQvWBr2UCHaldRkOYbpKc2eBr-sKMyKCK69aScvfyc&#x2F;kanban.png">kanban.png</a> | <a href="https:&#x2F;&#x2F;storage.www.wrike.com&#x2F;extfile&#x2F;eyJ0IjoiZmExIiwiYWxnIjoiSFMyNTYiLCJjdHkiOiJhcHBsaWNhdGlvblwvanNvbiJ9.eyJhIjoyMzQ5MjAzLCJyIjoiVVMiLCJleHAiOjMyNTAzNzA4ODAwLCJpYXQiOjE1Mzg3NTIwMTQsImp0aSI6IjcyNjY4OTgwIn0.KCpEi5-Mc1L3MnGsSV6MzNI3HZc0nJ82v10EydMV-5o&#x2F;folders.png">folders.png</a> | <a href="https:&#x2F;&#x2F;storage.www.wrike.com&#x2F;extfile&#x2F;eyJ0IjoiZmExIiwiYWxnIjoiSFMyNTYiLCJjdHkiOiJhcHBsaWNhdGlvblwvanNvbiJ9.eyJhIjoyMzQ5MjAzLCJyIjoiVVMiLCJleHAiOjMyNTAzNzA4ODAwLCJpYXQiOjE1Mzg3NTIwMTQsImp0aSI6IjcyNjY4OTgxIn0.onxu5ojsKYzQQPm7Tglx9INanVe9XS_esuYkPKfYf-Q&#x2F;show_subfolders.png">show_subfolders.png</a>
